Tchap/Managers/User/UserService.swift

```diff
@@ -109,21 +109,14 @@ final class UserService: NSObject, UserServiceType {
     }
     
     func isAccountDeactivated(for userId: String, completion: @escaping ((MXResponse<Bool>) -> Void)) -> MXHTTPOperation? {
-        return self.getUserInfo(for: userId) { (response) in
+        return self.getUsersInfo(for: [userId]) { (response) in
             switch response {
-            case .success(let userInfo):
-                completion(.success(userInfo.deactivated))
-            case .failure(let error):
-                completion(.failure(error))
-            }
-        }
-    }
-    
-    func isAccountExpired(for userId: String, completion: @escaping ((MXResponse<Bool>) -> Void)) -> MXHTTPOperation? {
-        return self.getUserInfo(for: userId) { (response) in
-            switch response {
-            case .success(let userInfo):
-                completion(.success(userInfo.expired))
+            case .success(let usersInfo):
+                if let deactivated = usersInfo[userId]?.deactivated {
+                    completion(.success(deactivated))
+                } else {
+                    completion(.success(false))
```

Comment:
I would prefer consider this case as a failure, don't you think so?
```suggestion
                    completion(.failure(UserServiceError.unknown))
```

Reply:
Applied